scripts/install_backend_python_libs.py

```diff
@@ -394,15 +393,38 @@ def _run_pip_command(cmd_parts):
         raise Exception('Error installing package')
 
 
+def pip_install_to_system(package, version):
+    """Installs third party libraries with pip to the user's system.
+
+    Note: These libraries are installed to the user's default system-wide
+    'site-packages' folder, not to a local Oppia third-party directory. This is
+    ONLY required in very specific cases where the development server scripts
+    require default libraries (When running another python script using
+    the shell, the call stack that is instantiated for that python script cannot
+    be edited by us; therefore, we have no control over which system paths, the
+    script visits when it looks for libraries and can only install those
+    necessary libraries to the default system path).
+
+    Please use pip_install instead if you are installing packages required for
```

Comment:
In general, please DO NOT use this method when installing packages required for oppia. Use pip_install instead.

Reply:
Done.